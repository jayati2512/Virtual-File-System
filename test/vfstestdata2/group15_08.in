createvfs vfs/vfs 1024
createvfs vfs2 1024
unmountvfs
mountvfs vfs2
addfile /home/ test1.txt /user/include/lastlog.h
makedir /home/ abc
makedir / home 		
makedir / user 		
makedir /home/ 		
makedir /home/ b/a 	
makedir /home/ a
movedir /home/test1.txt /user
movedir /home/ /home/a/
deletedir /home/abc/ 
listdir / 2 /tmp/a.txt 
addfile /home/ test1.txt /usr/include/lastlog.h
addfile / test2.png /usr/include/stab.h
listfile /test2.png /tmp/lout1.txt 
updatefile /home/test1.txt /usr/include/stdio.h
removefile /home/test1.txt
unmountvfs vfs2
