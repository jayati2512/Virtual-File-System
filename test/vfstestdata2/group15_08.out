createvfs_FAILURE INVALID_CHARACTER_IN_NAME
createvfs_SUCCESS
unmountvfs_FAILURE VFS_INSUFFICIENT_ARGUMENTS
mountvfs_SUCCESS
addfile_FAILURE DIRECTORY_DOES_NOT_EXIST
makedir_SUCCESS
makedir_FAILURE DIRECTORY_ALREADY_EXISTS
makedir_SUCCESS
makedir_FAILURE VFS_INSUFFICIENT_ARGUMENTS
makedir_FAILURE INVALID_CHARACTER_IN_DIRNAME
makedir_SUCCESS
movedir_FAILURE SOURCE_CANNOT_BE_FILE
movedir_FAILURE CANNOT_MOVE_PARENT_TO_CHILD_DIR
deletedir_FAILURE CANNOT_FIND_SPECIFIED_DIR
listdir_FAILURE INVALID_FLAG
addfile_SUCCESS
addfile_SUCCESS
listfile_FAILURE NOT_A_TEXT_FILE
updatefile_FAILURE EXTERNAL_FILE_TOO_LARGE
removefile_SUCCESS
unmount_SUCCESS
